charts/lily/templates/configmap-notifier.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if eq .Values.deploymentType "cluster" }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-lily-notifier-config
+  labels:
+    {{- include "sentinel-lily.allLabels" . | nindent 4 }}
+data:
+  {{- if .Values.cluster.notifier.inlineEnabled }}
+  config.toml: |
+  {{ .Values.cluster.notifier.inlineConfig | indent 2 }}
```

Comment:
Ran into this when using `daemon.inlineEnabled=true`.

```diff
- {{ .Values.cluster.notifier.inlineConfig | indent 2 }}
+ {{- .Values.cluster.notifier.inlineConfig | nindent 4 }}
```

```
# local.yaml
deploymentType: "cluster"
cluster:
  notifier:
    inlineEnabled: true
    inlineConfig: |
      [API]
        ListenAddress = "/ip4/0.0.0.0/tcp/1234/http"
      [Libp2p]
        ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]
        ConnMgrLow = 400
        ConnMgrHigh = 500
        ConnMgrGrace = "5m0s"
```

Current behavior
```
---
# Source: lily/templates/configmap-notifier.yaml

---
apiVersion: v1
kind: ConfigMap
metadata:
  name: coolcatmeow-lily-notifier-config
  labels:
    app.kubernetes.io/name: "lily-dev-mainnet-coolcatmeow"
    app.kubernetes.io/instance: coolcatmeow
    helm.sh/chart: lily-0.0.7
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/version: "latest"
    app.kubernetes.io/part-of: sentinel
    environment: dev
    network: mainnet
data:
  config.toml: |
    [API]
    ListenAddress = "/ip4/0.0.0.0/tcp/1234/http"
  [Libp2p]
    ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]
    ConnMgrLow = 400
    ConnMgrHigh = 500
    ConnMgrGrace = "5m0s"
```

```
helm template --debug coolcatmeow . --values local.yaml 2>&1 | egrep -B7 -A20 "^  name: coolcatmeow-lily-notifier-config"
```